amd/src/embed.js

```diff
@@ -45,7 +45,7 @@ define(['jquery', 'mod_hvp/communicator'], function($, H5PEmbedCommunicator) {
                 // When resize has been prepared tell parent window to resize.
                 H5PEmbedCommunicator.on('resizePrepared', function() {
                     H5PEmbedCommunicator.send('resize', {
-                        scrollHeight: iFrame.contentDocument.body.scrollHeight
+                        scrollHeight: iFrame.contentDocument.body.scrollHeight + ((globalThis.completion) ? 50 : 0)
```

Comment:
What is 50? Where does it come from? Could it be different in different Moodle versions or Moodle themes?

Reply:
That's the height of the activity completion information. Or maybe it's 60 ? i have to check again.

Reply:
Thanks for coming up with an approach to calculate the hight dynamically. That's looking good.